functions.php

```diff
@@ -38,110 +41,109 @@ function __construct() {
 		parent::__construct();
 	}
 
-	function enqueue_parent_styles() {
+	/**
+	 *
+	 */
+	protected function enqueue_parent_styles() {
```

Comment:
Guess I was overprotective :)
This needs to be public (as I said... I did not test it :p )

Reply:
Actually... they all need to be public otherwise they callback functionality of the wordpress actions won't work as expected.... so yes, change them to public and we are good to go.

Reply:
Yes I made it